### DISCUSSION TITLE
<audio>

### DISCUSSION THREAD

**Discussion by RobinManoli** [beercss/beercss#369] [General]
`<audio controls src='file.mp3'></audio>` looks good on computer but not on mobile (using chrome browser). It doesn't seem like it's adapted to mobile. So it would be nice if this tag could be adjusted for mobile within beer css.

**Comment by acroyear** [comment]
'controls' in the audio and video tag are supplied by the browser. at present the controls it presents are not things that can be externally styled by the official standard. Webkit (Chrome, Safari) supports [some selectors](https://stackoverflow.com/a/33948255), but it isn't an official standard so Firefox won't work.

The real solution is to leave controls off an handle it yourself. While there are 'recommendations' in M3X for what the controls can look like, it isn't something standardized and proscribed by the specification such that (in my opinion) it is in the scope of beercss to implement.

Here's some ideas - https://subfiresuite.com/_images/beercss/m3x-3%20play%20menu.png - of how you might assemble the buttons (a nav bar with a tags for prev/next etc, then a FAB for play/pause, and separately a slider for adjusting the time). Mine aren't coded to directly manipulate the audio tag as such; I have an API layer because i needed to also expose features to the media-session API or a 'remote control' (chromecast or an internal system I developed), so I wrote a layer that both media-session and these buttons can call to ensure the behavior's the same.
